18_testimonials/app_v2.js

```diff
@@ -0,0 +1,66 @@
+const buttons = document.querySelectorAll(".button")
+
+const cusImage = document.querySelector(".image")
+const cusName = document.querySelector(".name")
+const cusRating = document.querySelector(".rating")
+const cusTestimony = document.querySelector(".testimony")
+
+let i = 0
+const customers = []
+
+// Create new customer using constructor
+
+function Customer(image, name, rating, testimony) {
+    this.image = image
+    this.name = name
+    this.rating = rating
+    this.testimony = testimony
+}
+
+//Create customer using constructor function
+
+function createCustomer(image, name, rating, testimony) {
+
+    let customer = new Customer(image, name, rating, testimony)
```

Comment:
It's nice you learned about JS Constructors. They are a core part of VueJS for example.

However, in this case, I'd rather stick with an array of objects. But still, use the power of a fn to create new customers.

```js
function Customer(image, name, rating, testimony) {
 customers.push({ image, name, rating, testimony })
}
```